docs/source/suite.rst

```diff
@@ -16,21 +32,38 @@ contact-rich movements involving object-manipulation such as pen-twirling and ba
 
 myoFinger
 ==========
-Simplified and intuitive model (based on Xu et al. (2012)) of a 4 Degree of Freedom (DoF) finger
+Simplified and intuitive model (based on Xu et al. (2012)[https://ieeexplore.ieee.org/document/6290710]) of a 4 Degree of Freedom (DoF) finger
 which is actuated through a series of 5 simplified antagonistic muscle-tendon units.
 We also provide its robotic counterpart with simple torque actuators
 to facilitate the comparative investigation.
 
+Details of this model can be found at `myoFinger <https://github.com/MyoHub/myo_sim/tree/dbaecde82a18341661d4eb6d41928f26dc017bdd/finger>`__
+
+
 .. image:: images/myoFinger.png
   :height: 200
 
+========  =============================
+Acronym   Muscle
+========  =============================
+EXTN      Central Extensor
+adabR     Abduction Tendon(Right)
+adabL     Abduction Tendon(left)
+mflx      Pip Flexor
+dflx      DIP Flexor
+========  =============================
+
+
+
 .. _myoElbow:
 
 myoElbow
 ===========
 A model of 2 DoF human elbow joint -- based on OpenSim’s default testing arm model (Delp et al. (2007); Seth et al. (2018))
 -- and actuated using multiple agonist/antagonist pairs (3 flexors and 3 extensors).
 
+Details of this model can be found at `myoElbow <https://github.com/MyoHub/myo_sim/tree/dbaecde82a18341661d4eb6d41928f26dc017bdd/elbow>`__
```

Comment:
@Vittorio-Caggiano -- 
1. docs are pointing readers to the `myo_sim` repo. Do we want this? 
2. Additionally, they are linking to some weird commit. This should point to latest on `main` 
cc @raku-slyu

Reply:
updated and point the link to main now

Reply:
resolved following Vittorio's comment